Simplenote/src/androidTest/java/com/automattic/simplenote/BaseUITest.kt

```diff
@@ -0,0 +1,43 @@
+package com.automattic.simplenote
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import com.automattic.simplenote.models.Tag
+import com.automattic.simplenote.utils.TagUtils
+import com.automattic.simplenote.utils.TestBucket
+import org.junit.Before
+import java.security.SecureRandom
+
+open class BaseUITest {
+    private val charPool : List<Char> = ('a'..'z') + ('A'..'Z') + ('0'..'9')
+    private lateinit var application: SimplenoteTest
+    protected lateinit var tagsBucket: TestBucket<Tag>
+
+    @Before
+    fun setup() {
+        application = ApplicationProvider.getApplicationContext() as SimplenoteTest
+        tagsBucket = application.tagsBucket as TestBucket<Tag>
+        tagsBucket.clear()
+    }
+
+    protected fun getResourceString(id: Int): String? {
```

Comment:
Minor (🔍): The returned `String?` value can be defined as non-nullable, isn't it? Then, by doing so you will end up simplifying its usage (like removing the need for `... ?: ""`).

Reply:
Removed. The IDE also gave me the warning.

Reply:
IDEs, Android Studio and IntelliJ rocks, isn't it, I too noticed that because of the IDE, otherwise I wouldn't have noticed it! 😄 

PS: I do my reviews from the IDE as it helps me a lot with these kind of details.